INSTALL.md

```diff
@@ -0,0 +1,133 @@
+# Install Docker Compose
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
```

Comment:
Perhaps add a ⚠️ and mention that this is about Compose V1, and that we recommend installing V2 (and link to docs.docker.com)